extra/cap/cap.factor

```diff
@@ -24,7 +24,10 @@ IN: cap
     [ glReadPixels ] keep ;
 
 : screenshot ( window -- bitmap )
-    [ <image> t >>2x? ] dip
+    [ <image>
+        gl-scale-factor get-global
+        [ >integer 2 = [ >>2x? ] when* ] when*
```

Comment:
if you wanted it a little cleaner, this could probably be:

```factor
gl-scale-factor get-global [ 2.0 = >>2x? ] when*
```

Reply:
oh yeah, of course

Reply:
will `gl-scale-factor` always be `float`?